pkg/minishift/provisioner/minishift_provisioner.go

```diff
@@ -163,6 +161,12 @@ func (provisioner *MinishiftProvisioner) GenerateDockerOptions(dockerPort int) (
 	engineConfigTemplate := engineConfigTemplateCentOS
 	if provisioner.GetRedhatRelease() {
 		engineConfigTemplate = engineConfigTemplateRHEL
+	} else if provisioner.OsReleaseInfo.ID == "fedora" {
+		engineConfigTemplate = engineConfigTemplateFedora
+	}
+
+	if provisioner.GetRedhatRelease() {
```

Comment:
I think this is extra code as `provisioner.GetRedhatRelease()` is already handled.